docs/source/config.json

```diff
@@ -29,7 +29,7 @@
       "Subgraphs": "/building-supergraphs/subgraphs-overview",
       "Supported subgraph libraries": "/building-supergraphs/supported-subgraphs",
       "Router/gateway": "/building-supergraphs/router",
-      "Apollo Server subgraphs": "/building-supergraphs/subgraphs-apollo-server"
+      "Apollo Server subgraphs": "/apollo-server/subgraphs-apollo-server"
```

Comment:
Should we completely remove these links from the sidebar? Or redirect to the AS docs for now?

Reply:
It doesn't hurt much for them to remain here.. let's leave 'em. _However_, the URL needs to be absolute for cross-docset links

Reply:
```suggestion
      "Apollo Server subgraphs": "https://apollographql.com/docs/apollo-server/subgraphs-apollo-server"
```